src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -106,25 +113,25 @@ pub trait HummockVersionExt {
     fn get_object_ids(&self) -> Vec<u64>;
 }
 
+pub type BranchedSstInfo = HashMap<CompactionGroupId, /* SST Id */ HummockSstableId>;
```

Comment:
Why do we replace BTreeMap with HashMap? I think BTreeMap is faster.

Reply:
Emmmm ? 
HashMap is sure faster than BTreeMap...

Reply:
> Emmmm ? HashMap is sure faster than BTreeMap...

cc @BugenZhao He told me that `BTreeMap` is faster.

Reply:
When there're several or tens of entries, `BTreeMap` can be faster. 🥵

Reply:
OK, I would change it back.